components/CreateTripButton.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import { styled } from "styled-components";
+
+export default function NewTripButton() {
+  return (
+    <StyledButton>
+      <StyledButtonLink href="/FormPage">Add new trip</StyledButtonLink>
+    </StyledButton>
+  );
+}
+
+const StyledButton = styled.button`
+  border: 0.5px solid black;
+  font-size: 1em;
+  border-radius: 0.45rem;
+  padding: 0.25em 1em;
+  margin: 1em;
```

Comment:
You don't need a button around your link component. Please use CSS to make the link "look like" a button. You can then export the styled Link component directly like this:
```js
export default styled(Link)`
  ... Styling
`

Reply:
Thanks, makes the component smaller :)